Tests/TaggedTests/TaggedTests.swift

```diff
@@ -52,7 +52,9 @@ final class TaggedTests: XCTestCase {
   }
 
   func testLosslessStringConvertible() {
-    XCTAssertEqual(Tagged<Tag, Bool>(rawValue: true), Tagged<Tag, Bool>("true"))
+    // NB: This explicit `.init` shouldn't be necessary, but there seems to be a bug in Swift 5.
+    //     Filed here: https://bugs.swift.org/browse/SR-9752
+    XCTAssertEqual(Tagged<Tag, Bool>(rawValue: true), Tagged<Tag, Bool>.init("true"))
```

Comment:
Running into this compiler bug with the Xcode 10.2 beta. Filed a bug.

Reply:
I think this bug was fixed, so we should be able to get rid of this. I'm currently downloading Xcode 10.2 so can't test just yet